src/sentry/api/endpoints/organization_events_trends.py

```diff
@@ -261,9 +261,11 @@ def get_snql_function_aliases(trend_columns: TrendColumns, trend_type: str) -> d
                         if trend_type == IMPROVED
                         else aggregate_filter.operator
                     ),
-                    -1 * aggregate_filter.value.value
-                    if trend_type == IMPROVED
-                    else aggregate_filter.value.value,
+                    (
+                        -1 * aggregate_filter.value.value
+                        if trend_type == IMPROVED
+                        else aggregate_filter.value.value
```

Comment:
These changes (and the rest of the similar ones in this file) seem unrelated to `start_span`, maybe they got auto-formatted? Would consider splitting into separate PR, but not super important